plugins/Denizen/scripts/utility/player_restraining.dsc

```diff
@@ -0,0 +1,16 @@
+player_restraints:
+    type: world
+    debug: false
+    events:
+        on player walks flagged:util.nowalk:
+            - determine passively cancelled
+        on player steps on block flagged:util.nostep:
+            - determine passively cancelled
+        on player jumps flagged:util.nojump:
+            - determine passively cancelled
```

Comment:
Did you test this? player jumps isn't cancellable.
```suggestion
```

Reply:
oh shoot